packages/js/src/Modules/Verto/index.ts

```diff
@@ -21,7 +21,93 @@ export default class Verto extends BrowserSession {
+   * @param options.callerName Name to use as the caller ID name when dialing out to a destination.
+   * @param options.id The identifier of the call.
+   * @param options.localStream If set, the call will use this stream instead of retrieving a new one.
+   * @param options.localElement Overrides client's default `localElement`.
+   * @param options.remoteElement Overrides client's default `remoteElement`.
+   * @param options.iceServers Overrides client's default `iceServers`.
+   * @param options.audio Overrides client's default audio constraints. Defaults to `true`
+   * @param options.video Overrides client's default video constraints. Defaults to `false`
+   * @param options.useStereo Uses stereo audio instead of mono.
+   * @param options.micId `deviceId` to use as microphone. Overrides the client's default one.
+   * @param options.camId `deviceId` to use as webcam. Overrides the client's default one.
+   * @param options.speakerId `deviceId` to use as speaker. Overrides the client's default one.
+   * @param options.onNotification Overrides client's default `telnyx.notification` handler for this call.
+   *
+   * @return `Promise<Call>` A promise fulfilled with the new outbound `Call` object
+   * or rejected with the error.
+   *
+   * @examples
+   *
+   * Making an outbound call to `+1 856-444-0362` using default values from the client:
+   *
+   * if `options` is `null`.
+   * it will return the message error `You need to provide the options<CallOptions> object.`
+   *
+   * Using async/await:
+   *
+   * ```js
+   * const call = await client.newCall().catch(console.error)
+   * ```
+   *
+   * if `destinationNumber` is `null`.
+   * it will return the message error `destinationNumber is required.`
+   *
+   * Using async/await:
+   *
+   * ```js
+   * const options = {}
+   * const call = await client.newCall(options).catch(console.error)
+   * ```
+   *
+   * if `destinationNumber` is **not** `null`.
+   * it will make a call.
+   *
+   * Using async/await:
+   *
+   * ```js
+   * const options = { destinationNumber: '+18564440362' }
+   * const call = await client.newCall(options).catch(console.error)
+   * ```
+   *
+   *  To call a PSTN phone number:
+   *
+   * ```js
+   * const call = await client.newCall({
+   *  destinationNumber: '+18564440362',
+   *  callerNumber: '+15551231234' // Caller number is required when calling a phone number
+   * });
+   * ```
+   *
+   * To call a SIP address:
+   *
+   * ```js
+   * const call = await client.newCall({
+   *  destinationNumber: 'sip:example-sip-username@voip-provider.example.net'
+   * });
+   * ```
+   *
+   * If you are making calls from one Telnyx connection to another, you may specify just the SIP username:
+   *
+   * ```js
+   * const call = await client.newCall({
+   *  destinationNumber: 'telnyx-sip-username' // This is equivalent to 'sip:telnyx-sip-username@sip.telnyx.com'
+   * });
+   * ```
```

Comment:
```suggestion
   * Using async/await:
   *
   * ```js
   * const call = await client.newCall({
   *   destinationNumber: '+18564440362',
   *   callerNumber: '+15551231234'
   * });
   * ```
   *
   * Using ES6 `Promises`:
   *
   * ```js
   * client.newCall({
   *   destinationNumber: '+18564440362',
   *   callerNumber: '+15551231234'
   * }).then((call) => {
   *   // do something with the call
   * });
   * ```
   *
   * You can omit `callerNumber` when dialing a SIP address:
   *
   * ```js
   * const call = await client.newCall({
   *  destinationNumber: 'sip:example-sip-username@voip-provider.example.net'
   * });
   * ```
   *
   * If you are making calls from one Telnyx connection to another, you may specify just the SIP username:
   *
   * ```js
   * const call = await client.newCall({
   *  destinationNumber: 'telnyx-sip-username' // This is equivalent to 'sip:telnyx-sip-username@sip.telnyx.com'
   * });
   * ```
   *
   * ### Error handling
   *
   * If `options` or `destinationNumber` is not specified, it throw an error.
   *
   * ```js
   * client.newCall().catch(console.error);
   * // => `You need to provide the options<CallOptions> object.`
   * client.newCall({}).catch(console.error);
   * // => `destinationNumber is required.`
   * ```
```

Reply:
Nice